Dockerfile

```
@@ -0,0 +1,24 @@
+FROM python:buster as base
+RUN apt-get update
+RUN apt-get install -y curl
+ENV WEBAPP_FOLDER=/opt/todoapp
+RUN mkdir $WEBAPP_FOLDER
+WORKDIR $WEBAPP_FOLDER
+RUN curl -sSL https://install.python-poetry.org | python3 -
+ENV PATH="/root/.local/bin:$PATH:"
+ENV FLASK_APP=${WEBAPP_FOLDER}/todo_app/app.py
+EXPOSE 5000
+COPY poetry.toml .
+COPY pyproject.toml .
```

Again very minor, but we can combine these:
```suggestion
COPY poetry.toml pyproject.toml .
```